x-pack/plugins/session_view/public/components/session_view_toggle_options/index.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiPopover,
+  EuiSelectable,
+  EuiPopoverTitle,
+  EuiFlexItem,
+  EuiButtonIcon,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiSelectableOption } from '@elastic/eui/src/components/selectable/selectable_option';
+import { StateField } from '../../../common/types/session_view';
+import { useStyles } from './styles';
+
+export const SessionViewDisplayOptions = ({
+  onChange,
+  optionsStates,
+}: {
+  onChange: (vars: StateField) => void;
+  optionsStates: StateField;
+}) => {
+  const [isOptionDropdownOpen, setOptionDropdownOpen] = useState(false);
+
+  const styles = useStyles();
+
+  const timestampString = 'Timestamp';
```

Comment:
I would move these to outside the class above (just below imports) as const. e.g     const TIMESTAMP_OPTION_KEY / VERBOSE_MODE_OPTION_KEY